src/pybind/mgr/dashboard/controllers/feedback.py

```diff
@@ -0,0 +1,67 @@
+# # -*- coding: utf-8 -*-
+import os
+from collections import defaultdict
+from typing import Optional
+
+from .. import mgr
+from ..services import feedback
+from . import ApiController, ControllerDoc, RESTController
+from ..plugins import PLUGIN_MANAGER as PM
+
+@ApiController('/feedback', secure=False)
+@ControllerDoc("Feedback API", "Report")
+class Feedback(RESTController):
+    issueAPIkey = None
+
+    def __init__(self):  # pragma: no cover
+        super(Feedback, self).__init__()
+        self.cephTrackerClient = feedback.CephTrackerClient()
+
+    def create(self, project_id, tracker_id, subject, description, severity: Optional[str] = '3 - Minor'):
+        """
+        Create an issue.
+        :param component: The buggy ceph component.
+        :param title: The title of the issue.
+        :param description: The description of the issue.
+        """
+        feedbackModel = FeedbackModel(project_id, tracker_id, subject, description, severity)
+        return self.cephTrackerClient.create_issue(feedbackModel)
+
+    def get(self, issue_number):
+        """
+        Validate the issue tracker API given by user.
+        :param issueAPI: The issue tracker API access key.
+        """
+        return self.cephTrackerClient.get_issues(issue_number)
+
+
+class FeedbackModel:
```

Comment:
This should be moved to the domain/model layer. Feel free to create the aprroriate folder.
No need to suffix the class with `Model`.

Reply:
I've shifted this to a `model` folder but since it is imported in the controller which is also named as `Feedback`, I have to either add Model here or change the name of controller.

Reply:
@cypherean Yeah, you can change the name of the controller like this:
https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/controllers/home.py#L58